src/index.ts

```diff
@@ -117,7 +130,7 @@ export async function run(argv: string[]) {
     });
   }
 
-  process.once('SIGINT', async () => {
+  process.once('SIGINT', async () => {;
```

Comment:
Looks like an unintended typo.

Reply:
Fixed thks

Reply:
By the away i will check if is posible to do something with prompt